baseplate/experiments/providers/base.py

```diff
@@ -7,7 +7,7 @@
 class Experiment(object):
     """Base interface for experiment objects."""
 
-    def get_unique_id(self, **kwargs):
+    def get_unique_id(self, **_kwargs):
```

Comment:
I don't understand this change, there are other methods that _don't_ have this change.

Reply:
the arg is unused, _ prefix makes that explicit for the linter and humans

Reply:
It's not un-used by subclasses though

Reply:
This is just an base/interface, not something that is used directly.

Reply:
yeah, fair enough. it's probably because there's actually some code in there (it's not _just_ an abstract method. it's only abstract when a condition passes). i'll just pragma it away.